cms/templates/import.html

```diff
@@ -127,6 +191,19 @@ <h3 class="title">${_("Success")}</h3>
       </form>
     </article>
 
+  %if library:
+    <aside class="content-supplementary" role="complimentary">
+      <div class="bit">
+          <h3 class="title-3">${_("Why import a library?")}</h3>
+          <p>${_("You might want to update an existing library to a new version, or replace an existing library entirely. You might also have developed a library outside of Studio.")}</p>
+      </div>
+
+      <div class="bit">
+          <h3 class="title-3">${_("Warning: Importing while a library is in use")}</h3>
+          <p>${_("If you perform an import while your library is in use, any randomized content blocks which use it will need to be refreshed to take advantage of the new content.")}</p>
```

Comment:
@catong  - Ah ok I see. Apologies for the confusion. The suggested copy here makes sense to me.